### ISSUE TITLE
Add number of open documents to Documents List's Title Bar

### ISSUE THREAD

**Issue by Yaron10** [notepad-plus-plus/notepad-plus-plus#10395]
Currently the number of open documents is displayed in the Window dialog.
I find the Documents List much more useful and convenient.

**Suggestion:** 
Add the number of open documents to the Documents List's Title Bar.

Thank you.
***

Notepad++ v8.1.3.

**Comment by donho** [comment]
Window dialog is a modal dialog, so each access time we relaunch Window dialog, the title bar is re-displayed with the current number of total tabs opened.
Whereas the document list is a non-modal dockable dialog which could be displayed persistently while the document closed and opened - it means the implementation will be less easier since the title bar needs to be refreshed each time user closes/opens a document.

**Comment by mere-human** [comment]
> Whereas the document list is a non-modal dockable dialog which could be displayed persistently while the document closed and opened - it means the implementation will be less easier since the title bar needs to be refreshed each time user closes/opens a document.

Sure. It is not that easy, but I believe, it can be done.
There are other static text labels in Notepad++ that display up-to date information for things that change dynamically.
For example:
![image](https://user-images.githubusercontent.com/9664141/130042560-ed0a720f-6c0d-49b9-ac98-2991f3607342.png)

**Comment by Yaron10** [comment]
@donho,

Yes, I was aware of that difference.

How about displaying the number in the Window menu?

```
1: Test 1
2: Test 2
...
...
10: Test 10
(Total: 25) Window...
```
The last line tells the user: There are 25 open documents. Open the Window dialog to see/manage them.
***
@mere-human,

This is a modified version of my original suggestion.

**Comment by mere-human** [comment]
> How about displaying the number in the Window menu?

This is a bit unconventional.
Typically, the menu items are "commands" that perform some action or show some UI.
But in this case it's just a "label". Such thing is better to be displayed elsewhere (status bar, a section in Document List panel, etc).

**Comment by Yaron10** [comment]
> But in this case it's just a "label".

In the "modified version" it's not just a "label". It has a command.
But I agree, it's still a bit unconventional. OTOH, it's nice and useful. :)

**Comment by mere-human** [comment]
> In the "modified version" it's not just a "label". It has a command.

Ah, I see. You want to append the number of documents to the Window dialog menu item.

**Comment by Yaron10** [comment]
> Ah, I see. You want to append the number of documents to the Window dialog menu item.

Yes. I accepted your comment regarding my "original suggestion". :)
